node/src/BaseClient.ts

```diff
@@ -940,13 +946,28 @@ export class BaseClient {
             this.writeOrBufferCommandRequest(callbackIndex, command, route);
         });
     }
+    protected getValueToResolve<T>(resolveAns: T, decoder: boolean): T {
```

Comment:
Pls move the try-catch to the func. Placing the ```try-catch``` inside the function provides localized error handling and improves code readability by keeping error management close to where it occurs. This also avoids repetitive error wrapping around multiple function calls